### DISCUSSION TITLE
Taro3支持快应用编译吗

### DISCUSSION THREAD

**Discussion by Jer-X** [NervJS/taro#9482] [Q & A] [answered]
Taro3支持快应用编译吗

**Comment by luckyadam** [comment]
暂不支持

**Reply by xuliang2024** [reply]
为啥不支持

**Reply by Chen-jj** [reply]
因为适配成本高，但用户基数少，因此优先级不高，所以更希望是社区有同学或团队愿意推动

**Comment by yoonzm** [comment]
@luckyadam 请问目前有支持快应用的计划吗

**Reply by Chen-jj** [reply]
还没有。。

**Comment by Chen-jj** [comment, marked as answer]
最近有位大佬提交了支持快应用的 PR：

> Taro 3 支持开发快应用
> [PR](https://github.com/NervJS/taro/pull/9743)
> [RFC](https://github.com/NervJS/taro-rfcs/pull/12/files#diff-df4e0e698e7fad8ba3eb45ffd5d2f217c0262c047033b210e9d0d74bf5401623)

各位大佬有时间可以帮忙 Review 一下～🙏

**Reply by yoonzm** [reply]
向大佬学习
